modules/ROOT/partials/release-notes/6.0-release-notes-apis.adoc

```diff
@@ -55,41 +53,41 @@ The dialog API’s `setData` method now uses a deep merge algorithm.
 Consequently, it now supports partial nested objects.
 
 [[promise]]
-==== `success` and `failure` callbacks replaced by `Promise` in multiple APIs
+=== `success` and `failure` callbacks replaced by `Promise` in multiple APIs
 
- Previously, the AddOnManager, PluginManager, ScriptLoader, StyleSheetLoader, and ThemeManager APIs report resource loading status with `success` or `failure` callbacks.
+Previously, the AddOnManager, PluginManager, ScriptLoader, StyleSheetLoader, and ThemeManager APIs reported resource loading status with `success` or `failure` callbacks.
```

Comment:
Should these API names be surrounded by ` tags?

Reply:
Done